### DISCUSSION TITLE
Hide completed exception for "at least" measureable habits

### DISCUSSION THREAD

**Discussion by jrr96** [iSoron/uhabits#1922] [Feature Requests]
Hide completed is an amazing feature to de-clutter the screen when you have a lot of habits.

However, some of us also track some measurable habits through the day. Example: how much water did you drink today? I will not remember how much I drank by the end of the day so I note it as I drink.

This makes it slightly inconvenient because I have to toggle on and off the hide completed to update these habits.

A solution I can think of is to move the display of habits from global level to habit level and add three options: never hide, hide when filled, and hide when completed.

This should greatly improve the level of control the user has on which habits to display on screen.

**Comment by pzoechner** [comment]
Just came here to propose this feature.
I track about 40 habits, so I need to use the "Hide entered" feature to keep an overview of the tasks left in the day. "Measurable" habits like "Water", "Push-ups", etc. disappear after I enter the first value of the day. Adding more to the same habit requires me to deselect the "Hide entered" filter - hoping I don't forget about the habit.
